src/item.cpp

```diff
@@ -825,7 +825,41 @@ std::string Item::getDescription(const ItemType& it, int32_t lookDistance,
 				tmpSubType = item->getSubType();
 			}
 
-			s << ". " << (it.stackable && tmpSubType > 1 ? "They" : "It") << " can only be used with";
+			s << ". " << (it.stackable && tmpSubType > 1 ? "They" : "It") << " can only be used by ";
+
+			VocSpellMap vocMap = g_spells->getRuneSpell(it.id)->getVocMap();
```

Comment:
Removing a rune from spells.xml, reloading and looking at the rune would have been enough to trigger a null pointer dereference unless I'm missing something.